### DISCUSSION TITLE
Updating CEF version breaks build

### DISCUSSION THREAD

**Discussion by bonkyboo** [CefView/QCefView#387] [General]
Hi there! I'm hoping to use QCefView in a project and really appreciate the devs involved!

I have it building with VS2022 using Qt 5.15.2 or 6.7, using the CEF version that is in git: 113.3.1+g525fa10+chromium-113.0.5672.128

I updated the CEF version in QCefView\cmake\CefViewCoreConfig.cmake to the latest: 123.0.12+gb15e34c+chromium-123.0.6312.107
Then ran the generate-win-x86_64 script. 
Now I get a bunch of build errors when I try to build in VS or command line. Stuff like this:
```
CefViewDefaultRenderDelegate.cpp(67,36): error C2672: 'std::_Hash<std::_Umap_traits<_Kty,_Ty,std::_Uhash_compare<_
Kty,_Hasher,_Keyeq>,_Alloc,false>>::find': no matching overloaded function found

CefViewDefaultRenderDelegate.cpp(68,7): error C3536: 'it': cannot be used before it is initialized

CefViewDefaultRenderDelegate.cpp(155,38): error C2672: 'std::_Hash<std::_Umap_traits<_Kty,_Ty,std::_Uhash_compare<
_Kty,_Hasher,_Keyeq>,_Alloc,false>>::find': no matching overloaded function found

```

Does anyone know what I'm doing wrong? I tried using Qt 5.12, 5.15, and 6.7.

**Comment by tishion** [comment]
This should be the problems of c++ language standard

**Comment by DarthAhsoka** [comment]
![image](https://github.com/CefView/QCefView/assets/48704370/17c76e46-852a-49e1-adfa-185ebf7a1a44)
I also upgraded to the CEF version, but found that there were many compilation issues. I will see if it can be adapted, mainly because some hash_map key types have changed

**Reply by wangtao0505** [reply]
新版本的CEF的头文件中跟113版本的头文件函数名不一样，一个是小写on，一个是大写On
